modules/virt-enabling-preallocation-for-dv.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-enabling-preallocation-for-dv_{context}"]
+= Enabling preallocation for a data volume
+
+If preallocation is not globally enabled for the cluster, you can enable it for specific data volumes by including the `spec.preallocation` field in the data volume manifest. You can enable preallocation mode in either the web console or by using the OpenShift client (`oc`).
+
+Preallocation mode is supported for all CDI source types.
+
+.Procedure
+
+* Specify the `spec.preallocation` field in the data volume manifest:
++
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: preallocated-datavolume
+spec:
+  source: <1>
+    ...
+  pvc:
+    ...
+  preallocation: true <2>
+----
+<1> All CDI source types can use preallocation.
```

Comment:
Change:
<1> All CDI source types can use preallocation.
To:
<1> All CDI source types support preallocation.

Reply:
@kgoldbla If preallocation isn't valid for cloning is it worth making that distinction here?
I'm guessing that it won't throw an error but will ignore the preallocation field if the source is a pvc?

Reply:
Possibly. @tomob Can you please comment re @aburdenthehand question?

Reply:
Yes. The `preallocation` field will be ignored. I think it should be mentioned here.